contrib/cirrus/setup_environment.sh

```diff
@@ -177,6 +177,7 @@ case "$PODBIN_NAME" in
     *) die_unknown PODBIN_NAME
 esac
 
+make install.tools
```

Comment:
Instead of this...try rebasing on https://github.com/containers/podman/pull/8967

The `install.tools` target is run during each of the "Build for XYZ" tasks and the entire $GOPATH is cached for that VM.  The change in my PR simply restores that cache for the `alt_build` task.  I did not do this previously out of fear of polluting something (call it paranoia).